httpmw/Ip_cognito_auth_test.go

```diff
@@ -27,11 +30,22 @@ func (c *cognitoIdentityProviderClientMock) GetUser(input *cognitoidentityprovid
 	return args.Get(0).(*cognitoidentityprovider.GetUserOutput), args.Error(1)
 }
 
+func GetJWTtoken() (token string) {
+	claims := jwt.MapClaims{
+		"client_id": authTestCogntoClientId,
+	}
+
+	jwttoken := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	tokenString, _ := jwttoken.SignedString([]byte{})
+
+	return tokenString
```

Comment:
We should probably return token and the error here, if for some reason `SignedString` will fail, it's going to be hard for use to debug.